docs/lang/python.md

```diff
@@ -339,7 +339,7 @@ Success: no issues found in 1 source file
 
 ### 类型标注
 
-类型标注是自 Python 3.0 引入的特征，是添加类型提示的重要方法。例如这段代码就引入了类型标注，你可以通过调用 `circumference.__annotations__` 来查看函数中所有的类型标注。
+类型标注是自 Python 3.5 引入的特征 ([PEP 484](https://www.python.org/dev/peps/pep-0484/))，是添加类型提示的重要方法。例如这段代码就引入了类型标注，你可以通过调用 `circumference.__annotations__` 来查看函数中所有的类型标注。
```

Comment:
Strictly speaking, this feature was first introduced in Python 3.0 by [PEP 3107](https://www.python.org/dev/peps/pep-3107/), after [PEP 484](https://www.python.org/dev/peps/pep-0484/), it is basically available in [PEP 526](https://www.python.org/dev/peps/pep-0526/).